src/agent/main.c

```diff
@@ -54,18 +54,20 @@ static void get_opts(int argc, char *argv[]) {
                         break;
 
                 default:
-                        fprintf(stderr, "Unsupported option %c\n", opt);
+                        hirte_log_errorf("Unsupported option %c\n", opt);
```

Comment:
Same here, option parsing errors is imho straight stderr prints too.

Reply:
Changed.